slack-base/src/test/resources/conversations.replies.with_replies_and_new_fields.json

```diff
@@ -35,7 +35,19 @@
       "text": "Shutters shut and shutters and so shutters shut and shutters and so and so shutters and so shutters shut and so shutters shut and shutters and so.",
       "thread_ts": "1482960137.003543",
       "parent_user_id": "U061F7AUR",
-      "ts": "1483037603.017503"
+      "ts": "1483037603.017503",
+      "reactions": [
+        {
+          "name": "astonished",
+          "count": 3,
+          "users": [ "U061F7AUR", "U062F7AUR", "U063F7AUR"]
+        },
+        {
+          "name": "facepalm",
+          "count": 34,
+          "users": [ "U061F7AUR", "U062F7AUR", "U063F7AUR", "U064F7AUR", "U065F7AUR" ]
+        }
+      ]
```

Comment:
Thanks for adding these. 
Can we also please make sure that these reactions are parsed in the [corresponding integration test](https://github.com/HubSpot/slack-client/blob/d73e725448f8a0f59c5294639ca421ca87f05e32/slack-base/src/test/java/com/hubspot/slack/client/models/response/conversations/ConversationsRepliesResponseTest.java#L27-L36)?